src/coreclr/vm/jitinterface.cpp

```diff
@@ -2894,6 +2895,7 @@ void CEEInfo::embedGenericHandle(
                                                   pResolvedToken,
                                                   NULL,
                                                   pTemplateMD,
+                                                  (MethodDesc*)callerHandle,
```

Comment:
```suggestion
                                                  GetMethod(callerHandle),
```
This is the style used in this file instead of C-casts. (Fix all places.)